manifests/component_metadata.yaml

```
@@ -0,0 +1,4 @@
+releases:
+  - name: Kubeflow Trainer
+    version: "2.0.0"
+    repoUrl: https://github.com/kubeflow/trainer
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)